src/main/resources/templates/fragments/basic_search_form.html

```diff
@@ -125,4 +125,12 @@
             </select>
         </div>
     </div>
+    <div class="form-group row">
+        <label for="js-course-search-submit" class="col-sm-2 col-form-label"></label>
```

Comment:
This is most likely me not being familiar with HTML but do we need an empty label?

Reply:
I remember there being a particular reason for this regarding testing. All tests seemed to pass when label was removed however.

Reply:
I'd usually say that less code is better, so unless there is a good reason (i.e. it's correct html, it's useful for testing, etc.). I'd say either justify it or remove it. Otherwise the rest lgtm